jumanji/environments/packing/bin_pack/env_test.py

```diff
@@ -450,7 +450,6 @@ def test_bin_pack__optimal_policy_toy_instance(
             # assert not timestep.extras["invalid_action"]
             assert not jnp.any(timestep.extras["invalid_ems_from_env"])
         if timestep.extras is not None:
-            assert timestep.extras["volume_utilization"] == 1
             assert timestep.extras["ratio_packed_items"] == 1
```

Comment:
Why is this no longer 1? This should still pass for the toy instance right?

Reply:
No it's not supposed to pass, it was commented in the previous commits (and I was supposed to delete that line), I don't know in which commit I accidentally uncommented it. It's not supposed to pass for the toy instance because we have only 3 relatively small items which aren't enough to fill the bin.

Reply:
I had a bug with the action mask , I fixed it